python-checks/src/main/java/org/sonar/python/checks/UndeclaredNameUsageCheck.java

```diff
@@ -128,7 +128,7 @@ public void visitName(Name name) {
 
     @Override
     public void visitImportFrom(ImportFrom importFrom) {
-      hasWildcardImport |= importFrom.isWildcardImport();
+      hasUnresolvedWildcardImport |= (importFrom.isWildcardImport() && importFrom.hasUnresolvedWildcardImport());
```

Comment:
Is `importFrom.isWildcardImport()` actually still needed here?

Reply:
In the current implementation yes, because `importFrom.hasUnresolvedWildcardImport` returns `True` by default. However I will change its behaviour to avoid this `importFrom.isWildcardImport()`